README.md

```diff
@@ -1,6 +1,8 @@
 # Libxmljs
 [![Build Status](https://secure.travis-ci.org/polotek/libxmljs.svg?branch=master)](http://travis-ci.org/polotek/libxmljs)
 
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/cf862a98w7qsajpl?svg=true)](https://ci.appveyor.com/project/rchipka/libxmljs)
```

Comment:
This is the link to the polotek/libxmljs builds on my account. You, or anyone else for that matter, can create an AppVeyor account, point it to this repo, and it'll build and test exactly the same.

So it doesn't really matter which or how many different AppVeyor accounts point to this repo, as long as we link to one of them we'll have a badge and a link to the build/test logs.

You or I could create an AppVeyor account named "libxmljs" or "polotek" and then email the credentials to @polotek. That way he owns the AppVeyor account in the Readme.